README.md

```diff
@@ -145,15 +145,21 @@ The following table lists the global parameters supported by the chart and their
 | `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]` (does not add image pull secrets to deployed pods)  |
 | `global.mongodb.URL` | MongoDB URL | `mongodb://mongodb` |
 | `global.nats.URL` | NATS URL | `nats://nats:4222` |
+| `global.storage` | Storage type for artifacts (available types: `aws` and `azure`) | `aws` |
 | `global.s3.AWS_URI` | AWS S3 / MinIO URI | value from `global.url` |
 | `global.s3.AWS_EXTERNAL_URI` | External AWS S3 / MinIO URI | `null` |
-| `global.s3.AWS_BUCKET` | AWS S3 / MinIO bucket | `minio-hosted-mender-artifacts` |
+| `global.s3.DEPLOYMENTS_STORAGE_BUCKET` | AWS S3 / MinIO bucket | `minio-hosted-mender-artifacts` |
 | `global.s3.AWS_REGION` | AWS S3 region | `us-east-1` |
 | `global.s3.AWS_ACCESS_KEY_ID` | AWS S3 / MinIO key ID. An empty value will use credentials from the shared AWS credentials. | `myaccesskey` |
 | `global.s3.AWS_SECRET_ACCESS_KEY` | AWS S3 / MinIO access key | `mysecretkey` |
 | `global.s3.AWS_SERVICE_ACCOUNT_NAME` | Use K8s service account instead of `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to access the bucket. | `""` |
 | `global.s3.AWS_FORCE_PATH_STYLE` | Set the AWS S3 URI style to path | `true` |
 | `global.s3.AWS_TAG_ARTIFACT` | Tag the artifact in the AWS S3 storage service with the tenant ID | `true` |
+| `global.azure.AZURE_AUTH_CONNECTION_STRING` | Azure Blob Storage connection string | `mysecretconnstring` |
+| `global.azure.AZURE_AUTH_SHARED_KEY_ACCOUNT_NAME` | Azure Blob Storage shared key account name | `""` |
+| `global.azure.AZURE_AUTH_SHARED_KEY_ACCOUNT_KEY` | Azure Blob Storage shared key account key | `myaccountkey` |
+| `global.azure.AZURE_AUTH_SHARED_KEY_ACCOUNT_URI` | Azure Blob Storage shared key URI | `""` |
+| `global.azure.DEPLOYMENTS_STORAGE_BUCKET` | Azure Blob Storage container | `minio-hosted-mender-artifacts` |
```

Comment:
I would simply call it for what it is, although it maps to the same parameter.
```suggestion
| `global.azure.CONTAINER_NAME` | Azure Blob Storage container | `minio-hosted-mender-artifacts` |
```
Since azure is already in the object path, I would strip the `AZURE_` prefix from the other attributes as well.